pkgs/by-name/hp/hplip/package.nix

```diff
@@ -161,8 +161,11 @@ python3Packages.buildPythonApplication {
   postPatch = ''
     # https://github.com/NixOS/nixpkgs/issues/44230
     substituteInPlace createPPD.sh \
-      --replace ppdc "${cups}/bin/ppdc" \
-      --replace "gzip -c" "gzip -cn"
+      --replace-fail ppdc "${cups}/bin/ppdc" \
```

Comment:
In my opinion, it would be better to put changes like `--replace-fail` and `with lib;` below (both of which I highly endorse :+1: ) into separate commits, to ease bisection and reversion.  There is currently a [discussion about formal policy regarding commit separation](https://github.com/NixOS/nixpkgs/pull/400934).  But this is just for your information -- I would not consider this a blocker (and it wouldn't be my call anyway since I'm not a committer).